kolibri/core/assets/src/api-resources/task.js

```diff
@@ -89,6 +127,12 @@ export default new Resource({
     });
   },
 
+  deleteBulkChannels(channelIds) {
```

Comment:
Have this one take an object argument `{ channel_ids }` so its consistent with other functions